recipes/parallelproj/build_cuda.sh

```diff
@@ -0,0 +1,2 @@
+cmake -GNinja -DCMAKE_INSTALL_PREFIX=${PREFIX} -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LIBDIR=lib ${CMAKE_ARGS} -DCMAKE_CUDA_ARCHITECTURES=all -DSKIP_DOCS=TRUE -DSKIP_OPENMP_LIB=TRUE ${SRC_DIR}
+cmake --build . --target install --verbose
```

Comment:
Good practice is generally to not build directly in-tree (use build folder), quit on error, and use linebreaks, i.e.

```
#!/bin/bash
set -ex

mkdir build
cd build

cmake -G Ninja \ 
    -DALSO \
    -DPLEASE \
    -DUSE \
    -DLINEBREAKS \
    -DFOR \
    -DREADABILITY \
    [other options] \
    ..

cmake --build . --target install --verbose 
```

Reply:
build files have been modified accordingly